@koriym 

- my.cnfが存在しないことでエラーになっているようです。


<details><summary>詳細はこちら</summary><div>

- エラー
```bash

adachi in 🌐 vmmac in homebrew-brewworks on  multiple-port via 💎 v2.6.10 took 7m12s
❯ source brewworks env

adachi in 🌐 vmmac in homebrew-brewworks on  multiple-port via 💎 v2.6.10
❯

adachi in 🌐 vmmac in homebrew-brewworks on  multiple-port via 💎 v2.6.10
❯

adachi in 🌐 vmmac in homebrew-brewworks on  multiple-port via 💎 v2.6.10
❯ brewworks start
Starting php-fpm services...
Starting php-fpm with custom config on port 9000...
Starting mysql services...
Starting mysql with custom config on port 3306...
Starting mysql with custom config on port 3307...
Starting redis services...
Starting redis-server with custom config on port 6379...
Starting redis-server with custom config on port 6380...
Starting memcached services...
Starting memcached with custom config on port 11211...
Starting memcached with custom config on port 11212...
Starting nginx services...
Starting nginx with custom config on port 8080...
Starting httpd services...
Starting httpd with custom config on port 8082...
Services started with these settings:
/opt/homebrew/Cellar/brewworks/1.0.0/brewworks/config/php-fpm_9000.conf
/opt/homebrew/Cellar/brewworks/1.0.0/brewworks/config/php.ini
/opt/homebrew/Cellar/brewworks/1.0.0/brewworks/config/my_3306.cnf
/opt/homebrew/Cellar/brewworks/1.0.0/brewworks/config/my_3307.cnf
/opt/homebrew/Cellar/brewworks/1.0.0/brewworks/config/redis_6379.conf
/opt/homebrew/Cellar/brewworks/1.0.0/brewworks/config/redis_6380.conf
/opt/homebrew/Cellar/brewworks/1.0.0/brewworks/config/memcached_11211.conf
/opt/homebrew/Cellar/brewworks/1.0.0/brewworks/config/memcached_11212.conf
/opt/homebrew/Cellar/brewworks/1.0.0/brewworks/config/nginx_8080.conf
/opt/homebrew/Cellar/brewworks/1.0.0/brewworks/config/httpd_8082.conf
The web document root: /opt/homebrew/Cellar/brewworks/1.0.0/brewworks/public

adachi in 🌐 vmmac in homebrew-brewworks on  multiple-port via 💎 v2.6.10
❯ <br />
<b>Warning</b>:  PHP Startup: Unable to load dynamic library 'xdebug.so' (tried: /opt/homebrew/Cellar/php/8.3.8/pecl/20230831/xdebug.so (dlopen(/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/xdebug.so, 0x0009): tried: '/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/xdebug.so' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/xdebug.so' (no such file), '/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/xdebug.so' (no such file), '/opt/homebrew/lib/php/pecl/20230831/xdebug.so' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/lib/php/pecl/20230831/xdebug.so' (no such file), '/opt/homebrew/lib/php/pecl/20230831/xdebug.so' (no such file)), /opt/homebrew/Cellar/php/8.3.8/pecl/20230831/xdebug.so.so (dlopen(/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/xdebug.so.so, 0x0009): tried: '/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/xdebug.so.so' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/xdebug.so.so' (no such file), '/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/xdebug.so.so' (no such file), '/opt/homebrew/lib/php/pecl/20230831/xdebug.so.so' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/lib/php/pecl/20230831/xdebug.so.so' (no such file), '/opt/homebrew/lib/php/pecl/20230831/xdebug.so.so' (no such file))) in <b>Unknown</b> on line <b>0</b><br />
<br />
<b>Warning</b>:  PHP Startup: Unable to load dynamic library 'pcov.so' (tried: /opt/homebrew/Cellar/php/8.3.8/pecl/20230831/pcov.so (dlopen(/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/pcov.so, 0x0009): tried: '/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/pcov.so' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/pcov.so' (no such file), '/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/pcov.so' (no such file), '/opt/homebrew/lib/php/pecl/20230831/pcov.so' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/lib/php/pecl/20230831/pcov.so' (no such file), '/opt/homebrew/lib/php/pecl/20230831/pcov.so' (no such file)), /opt/homebrew/Cellar/php/8.3.8/pecl/20230831/pcov.so.so (dlopen(/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/pcov.so.so, 0x0009): tried: '/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/pcov.so.so' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/pcov.so.so' (no such file), '/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/pcov.so.so' (no such file), '/opt/homebrew/lib/php/pecl/20230831/pcov.so.so' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/lib/php/pecl/20230831/pcov.so.so' (no such file), '/opt/homebrew/lib/php/pecl/20230831/pcov.so.so' (no such file))) in <b>Unknown</b> on line <b>0</b><br />
<br />
<b>Warning</b>:  PHP Startup: Unable to load dynamic library 'redis.so' (tried: /opt/homebrew/Cellar/php/8.3.8/pecl/20230831/redis.so (dlopen(/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/redis.so, 0x0009): tried: '/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/redis.so' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/redis.so' (no such file), '/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/redis.so' (no such file), '/opt/homebrew/lib/php/pecl/20230831/redis.so' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/lib/php/pecl/20230831/redis.so' (no such file), '/opt/homebrew/lib/php/pecl/20230831/redis.so' (no such file)), /opt/homebrew/Cellar/php/8.3.8/pecl/20230831/redis.so.so (dlopen(/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/redis.so.so, 0x0009): tried: '/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/redis.so.so' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/redis.so.so' (no such file), '/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/redis.so.so' (no such file), '/opt/homebrew/lib/php/pecl/20230831/redis.so.so' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/lib/php/pecl/20230831/redis.so.so' (no such file), '/opt/homebrew/lib/php/pecl/20230831/redis.so.so' (no such file))) in <b>Unknown</b> on line <b>0</b><br />
<br />
<b>Warning</b>:  PHP Startup: Unable to load dynamic library 'memcached.so' (tried: /opt/homebrew/Cellar/php/8.3.8/pecl/20230831/memcached.so (dlopen(/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/memcached.so, 0x0009): tried: '/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/memcached.so' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/memcached.so' (no such file), '/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/memcached.so' (no such file), '/opt/homebrew/lib/php/pecl/20230831/memcached.so' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/lib/php/pecl/20230831/memcached.so' (no such file), '/opt/homebrew/lib/php/pecl/20230831/memcached.so' (no such file)), /opt/homebrew/Cellar/php/8.3.8/pecl/20230831/memcached.so.so (dlopen(/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/memcached.so.so, 0x0009): tried: '/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/memcached.so.so' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/memcached.so.so' (no such file), '/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/memcached.so.so' (no such file), '/opt/homebrew/lib/php/pecl/20230831/memcached.so.so' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/lib/php/pecl/20230831/memcached.so.so' (no such file), '/opt/homebrew/lib/php/pecl/20230831/memcached.so.so' (no such file))) in <b>Unknown</b> on line <b>0</b><br />
<br />
<b>Warning</b>:  PHP Startup: Unable to load dynamic library 'xdebug.so' (tried: /opt/homebrew/Cellar/php/8.3.8/pecl/20230831/xdebug.so (dlopen(/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/xdebug.so, 0x0009): tried: '/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/xdebug.so' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/xdebug.so' (no such file), '/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/xdebug.so' (no such file), '/opt/homebrew/lib/php/pecl/20230831/xdebug.so' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/lib/php/pecl/20230831/xdebug.so' (no such file), '/opt/homebrew/lib/php/pecl/20230831/xdebug.so' (no such file)), /opt/homebrew/Cellar/php/8.3.8/pecl/20230831/xdebug.so.so (dlopen(/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/xdebug.so.so, 0x0009): tried: '/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/xdebug.so.so' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/xdebug.so.so' (no such file), '/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/xdebug.so.so' (no such file), '/opt/homebrew/lib/php/pecl/20230831/xdebug.so.so' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/lib/php/pecl/20230831/xdebug.so.so' (no such file), '/opt/homebrew/lib/php/pecl/20230831/xdebug.so.so' (no such file))) in <b>Unknown</b> on line <b>0</b><br />
<br />
<b>Warning</b>:  PHP Startup: Unable to load dynamic library 'pcov.so' (tried: /opt/homebrew/Cellar/php/8.3.8/pecl/20230831/pcov.so (dlopen(/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/pcov.so, 0x0009): tried: '/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/pcov.so' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/pcov.so' (no such file), '/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/pcov.so' (no such file), '/opt/homebrew/lib/php/pecl/20230831/pcov.so' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/lib/php/pecl/20230831/pcov.so' (no such file), '/opt/homebrew/lib/php/pecl/20230831/pcov.so' (no such file)), /opt/homebrew/Cellar/php/8.3.8/pecl/20230831/pcov.so.so (dlopen(/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/pcov.so.so, 0x0009): tried: '/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/pcov.so.so' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/pcov.so.so' (no such file), '/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/pcov.so.so' (no such file), '/opt/homebrew/lib/php/pecl/20230831/pcov.so.so' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/lib/php/pecl/20230831/pcov.so.so' (no such file), '/opt/homebrew/lib/php/pecl/20230831/pcov.so.so' (no such file))) in <b>Unknown</b> on line <b>0</b><br />
<br />
<b>Warning</b>:  PHP Startup: Unable to load dynamic library 'redis.so' (tried: /opt/homebrew/Cellar/php/8.3.8/pecl/20230831/redis.so (dlopen(/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/redis.so, 0x0009): tried: '/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/redis.so' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/redis.so' (no such file), '/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/redis.so' (no such file), '/opt/homebrew/lib/php/pecl/20230831/redis.so' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/lib/php/pecl/20230831/redis.so' (no such file), '/opt/homebrew/lib/php/pecl/20230831/redis.so' (no such file)), /opt/homebrew/Cellar/php/8.3.8/pecl/20230831/redis.so.so (dlopen(/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/redis.so.so, 0x0009): tried: '/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/redis.so.so' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/redis.so.so' (no such file), '/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/redis.so.so' (no such file), '/opt/homebrew/lib/php/pecl/20230831/redis.so.so' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/lib/php/pecl/20230831/redis.so.so' (no such file), '/opt/homebrew/lib/php/pecl/20230831/redis.so.so' (no such file))) in <b>Unknown</b> on line <b>0</b><br />
<br />
<b>Warning</b>:  PHP Startup: Unable to load dynamic library 'memcached.so' (tried: /opt/homebrew/Cellar/php/8.3.8/pecl/20230831/memcached.so (dlopen(/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/memcached.so, 0x0009): tried: '/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/memcached.so' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/memcached.so' (no such file), '/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/memcached.so' (no such file), '/opt/homebrew/lib/php/pecl/20230831/memcached.so' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/lib/php/pecl/20230831/memcached.so' (no such file), '/opt/homebrew/lib/php/pecl/20230831/memcached.so' (no such file)), /opt/homebrew/Cellar/php/8.3.8/pecl/20230831/memcached.so.so (dlopen(/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/memcached.so.so, 0x0009): tried: '/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/memcached.so.so' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/memcached.so.so' (no such file), '/opt/homebrew/Cellar/php/8.3.8/pecl/20230831/memcached.so.so' (no such file), '/opt/homebrew/lib/php/pecl/20230831/memcached.so.so' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/lib/php/pecl/20230831/memcached.so.so' (no such file), '/opt/homebrew/lib/php/pecl/20230831/memcached.so.so' (no such file))) in <b>Unknown</b> on line <b>0</b><br />
[18-Jun-2024 20:20:25] NOTICE: fpm is running, pid 7915
[18-Jun-2024 20:20:25] NOTICE: ready to handle connections
nginx: [emerg] "server" directive is not allowed here in /opt/homebrew/Cellar/brewworks/1.0.0/brewworks/config/nginx_8080.conf:1
2024-06-18T13:20:26.6NZ mysqld_safe Logging to '/opt/homebrew/Cellar/brewworks/1.0.0/brewworks/logs/mysql_3307_error.log'.
2024-06-18T13:20:26.6NZ mysqld_safe Logging to '/opt/homebrew/Cellar/brewworks/1.0.0/brewworks/logs/mysql_3306_error.log'.
2024-06-18T13:20:26.6NZ mysqld_safe Starting mysqld daemon with databases from /opt/homebrew/Cellar/brewworks/1.0.0/brewworks/mysql_0
2024-06-18T13:20:26.6NZ mysqld_safe Starting mysqld daemon with databases from /opt/homebrew/Cellar/brewworks/1.0.0/brewworks/mysql_1


adachi in 🌐 vmmac in homebrew-brewworks on  multiple-port via 💎 v2.6.10
❯ brew test brewworks.rb
Warning: test is a developer command, so Homebrew's
developer mode has been automatically turned on.
To turn developer mode off, run:
  brew developer off

Fetching gem metadata from https://rubygems.org/.......
Fetching public_suffix 5.0.5
Fetching bindata 2.5.0
Fetching minitest 5.23.1
Fetching plist 3.7.1
Installing bindata 2.5.0
Installing public_suffix 5.0.5
Fetching ruby-macho 4.1.0
Fetching sorbet-runtime 0.5.11430
Installing minitest 5.23.1
Fetching warning 1.4.0
Installing plist 3.7.1
Fetching addressable 2.8.6
Installing ruby-macho 4.1.0
Fetching elftools 1.3.1
Installing sorbet-runtime 0.5.11430
Installing warning 1.4.0
Installing addressable 2.8.6
Installing elftools 1.3.1
Fetching patchelf 1.5.1
Installing patchelf 1.5.1
Bundle complete! 38 Gemfile dependencies, 11 gems now installed.
Bundled gems are installed into `../../../../opt/homebrew/Library/Homebrew/vendor/bundle`
==> Testing brewworks
==> php -v
PHP 8.3.8 (cli) (built: Jun  4 2024 14:53:17) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.3.8, Copyright (c) Zend Technologies
    with Zend OPcache v8.3.8, Copyright (c), by Zend Technologies
==> mysql --version
Last 15 lines from /Users/adachi/Library/Logs/Homebrew/brewworks/test.02.mysql:
2024-06-18 20:21:15 +0700

mysql --version

Error: brewworks: failed
Warning: Removed Sorbet lines from backtrace!
Rerun with `--verbose` to see the original backtrace
An exception occurred within a child process:
  BuildError: Failed executing: mysql --version
/opt/homebrew/Library/Homebrew/formula.rb:3041:in `block in system'
/opt/homebrew/Library/Homebrew/formula.rb:2977:in `open'
/opt/homebrew/Library/Homebrew/formula.rb:2977:in `system'
/Users/adachi/git/homebrew-brewworks/brewworks.rb:372:in `test_service'
/Users/adachi/git/homebrew-brewworks/brewworks.rb:378:in `block in <class:Brewworks>'
/opt/homebrew/Library/Homebrew/formula.rb:2783:in `block (3 levels) in run_test'
/opt/homebrew/Library/Homebrew/extend/kernel.rb:528:in `with_env'
/opt/homebrew/Library/Homebrew/formula.rb:2782:in `block (2 levels) in run_test'
/opt/homebrew/Library/Homebrew/formula.rb:1188:in `with_logging'
/opt/homebrew/Library/Homebrew/formula.rb:2781:in `block in run_test'
/opt/homebrew/Library/Homebrew/mktemp.rb:75:in `block in run'
/opt/homebrew/Library/Homebrew/mktemp.rb:75:in `chdir'
/opt/homebrew/Library/Homebrew/mktemp.rb:75:in `run'
/opt/homebrew/Library/Homebrew/formula.rb:3092:in `mktemp'
/opt/homebrew/Library/Homebrew/formula.rb:2775:in `run_test'
/opt/homebrew/Library/Homebrew/test.rb:46:in `block in <main>'
/opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.3.3/lib/ruby/3.3.0/timeout.rb:186:in `block in timeout'
/opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.3.3/lib/ruby/3.3.0/timeout.rb:41:in `handle_timeout'
/opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.3.3/lib/ruby/3.3.0/timeout.rb:195:in `timeout'
/opt/homebrew/Library/Homebrew/test.rb:50:in `<main>'
brew test brewworks.rb # 9.69s

adachi in 🌐 vmmac in homebrew-brewworks on  multiple-port via 💎 v2.6.10 took 9s
❯
```

- 調査

```bash
#2024-06-18 20:51:57 adachi@vmmac.local:~/git/homebrew-brewworks multiple-port ghgo.jp/koriym/homebrew-brewworks/ 91694fb Untracked:1
! type mysql
mysql is an alias for /opt/homebrew/opt/mysql@8.0/bin/mysql --defaults-file=/opt/homebrew/Cellar/brewworks/1.0.0/brewworks/config/my.cnf


#2024-06-18 20:52:38 adachi@vmmac.local:~/git/homebrew-brewworks multiple-port ghgo.jp/koriym/homebrew-brewworks/ 91694fb Untracked:1
 mysql
mysql: [ERROR] Failed to open required defaults file: /opt/homebrew/Cellar/brewworks/1.0.0/brewworks/config/my.cnf
mysql: [ERROR] Fatal error in defaults handling. Program aborted!


#2024-06-18 20:52:40 adachi@vmmac.local:~/git/homebrew-brewworks multiple-port ghgo.jp/koriym/homebrew-brewworks/ 91694fb Untracked:1
! ls /opt/homebrew/Cellar/brewworks/1.0.0/brewworks/config/my.cnf
ls: /opt/homebrew/Cellar/brewworks/1.0.0/brewworks/config/my.cnf: No such file or directory
#2024-06-18 20:52:48 adachi@vmmac.local:~/git/homebrew-brewworks multiple-port ghgo.jp/koriym/homebrew-brewworks/ 91694fb Untracked:1
```


</div></details>

--- a/README.md
+++ b/README.md
@@ -84,7 +84,7 @@ This command will start all the services defined in your configuration file.
 . brewworks env
 ````
 
-This environment variable setting causes `php` and `mysql` to point to the specified version of the binary. The `mysql` client contains project-specific `my.conf` settings so you don't have to specify a port number or user name for every connection. A `mysql` command with no arguments will connect to the project-specific database.
+This environment variable setting causes `php` and `mysql` to point to the specified version of the binary. `mysql -P 3306` - You must specify the port in this way to connect. (username and hostname can be omitted)
 
 ## Project Structure
 

--- a/brewworks.rb
+++ b/brewworks.rb
@@ -211,7 +211,7 @@ class Brewworks < Formula
       #{HOMEBREW_PREFIX}/opt/httpd/bin:
       #{HOMEBREW_PREFIX}/opt/node/bin:$PATH"
       export PHP_INI_SCAN_DIR="#{config_dir}"
-      alias mysql="#{HOMEBREW_PREFIX}/opt/mysql@#{MYSQL_VERSION}/bin/mysql --defaults-file=#{config_dir}/my.cnf"
+      alias mysql="#{HOMEBREW_PREFIX}/opt/mysql@#{MYSQL_VERSION}/bin/mysql"
     SCRIPT
 
     chmod "+x", script_dir/"env.sh"

--- a/brewworks.rb
+++ b/brewworks.rb
@@ -4,6 +4,7 @@ class Brewworks < Formula
   url "file:///dev/null"
   version "1.0.0"
 
+  # -- Begin Configuration Section --
   PROJECT_NAME = "brewworks"
   PHP_VERSION = "8.3"
   MYSQL_VERSION = "8.0"
@@ -26,8 +27,10 @@ class Brewworks < Formula
     httpd: [8082]
   }
   PHP_EXTENSIONS = ["xdebug", "pcov", "redis", "memcached"]
+  # -- End Configuration Section --
 
   DEPENDENCIES.each { |dep| depends_on dep }
+
 
   def install
     project_dir = Pathname.new(prefix)/PROJECT_NAME

--- a/brewworks.rb
+++ b/brewworks.rb
@@ -204,6 +204,7 @@ class Brewworks < Formula
     (script_dir/"env.sh").write <<~SCRIPT
       #!/bin/bash
       export PATH="#{HOMEBREW_PREFIX}/opt/php@#{PHP_VERSION}/bin:
+      export PATH="#{HOMEBREW_PREFIX}/opt/mysql@#{MYSQL_VERSION}/bin:
       #{HOMEBREW_PREFIX}/opt/mysql@#{MYSQL_VERSION}/bin:
       #{HOMEBREW_PREFIX}/opt/redis/bin:
       #{HOMEBREW_PREFIX}/opt/memcached/bin:
@@ -211,7 +212,6 @@ class Brewworks < Formula
       #{HOMEBREW_PREFIX}/opt/httpd/bin:
       #{HOMEBREW_PREFIX}/opt/node/bin:$PATH"
       export PHP_INI_SCAN_DIR="#{config_dir}"
-      alias mysql="#{HOMEBREW_PREFIX}/opt/mysql@#{MYSQL_VERSION}/bin/mysql"
     SCRIPT
 
     chmod "+x", script_dir/"env.sh"

